site/docs/capabilities/llm-integrations/supported-endpoints.md

```diff
@@ -191,6 +191,40 @@ curl -H "Content-Type: application/json" \
   $GATEWAY_URL/v1/images/generations
 ```
 
+### Rerank
+
+**Endpoint:** `POST /cohere/v2/rerank`
+
+**Status:** ✅ Fully Supported
+
+**Description:** Rerank a list of documents for a given query to return relevance scores and an ordered list. Cohere-compatible API.
+
+**Features:**
+
+- ✅ Single-query document reranking
+- ✅ Model selection via request body or `x-ai-eg-model` header
+- ✅ Token usage tracking and cost calculation
+- ✅ Provider fallback and load balancing
+
+**Supported Providers:**
+
+- Cohere
```

Comment:
could you mention the Cohere compatible providers here like vllm like embeddings endpoint